keadm/cmd/keadm/app/cmd/reset.go

```diff
@@ -74,12 +76,21 @@ func NewKubeEdgeReset(out io.Writer, reset *types.ResetOptions) *cobra.Command {
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// Tear down cloud node. It includes
-			// 1. killing cloudcore process
+			// 1. kill cloudcore/edgecore process.
+			// For edgecore, don't delete node from K8S
+			if err := TearDownKubeEdge(IsEdgeNode, reset.Kubeconfig); err != nil {
+				return err
+			}
+
+			// 2. remove containers managed by KE
+			_, _ = fmt.Fprintln(out, "[Test] Test here!")
+			if err2 := RemoveContainers(utilsexec.New()); err2 != nil {
```

Comment:
should not delete containers when using `keadm reset` in cloud.

Reply:
Thanks for your correction! Is this correct?

```go
if IsEdgeNode {
	if err2 := RemoveContainers(utilsexec.New()); err2 != nil {
		return err2
	}
}
```

Reply:
yes :), need to judge if on edgenode